### PR TITLE
Make codemirror compatible with scss

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes
- Remove error on template scss file edit
- Beautify the contents of an scss file

Info: https://codemirror.net/mode/css/scss.html

### Testing Instructions
Make sure you have some scss files in one of your template,
Try to edit this file, observe the console and the text inside the editor

### Before
screen shot 2017-03-11 at 11 44 38

### After
screen shot 2017-03-11 at 11 45 01

### Documentation Changes Required
None

### CODE REVIEW

File: plugins/editors/codemirror/codemirror.php [hunk 1]
```diff
@@ -31,6 +31,7 @@ class PlgEditorCodemirror extends JPlugin
 	 * @var array
 	 */
 	protected $modeAlias = array(
+			'scss' => 'css',
```
Comment (brianteeman) [inline plugins/editors/codemirror/codemirror.php]: the ocd in me says that this should go at the end of the array to maintin alphabetical order/

Comment (dgt41) [inline plugins/editors/codemirror/codemirror.php]: Makes sense...

Comment (brianteeman) [inline plugins/editors/codemirror/codemirror.php]: Of course it does ;) 

thanks

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 4e1763de2b340341e14cd7694bafe26ef4e46806<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14485">issues.joomla.org/tracker/joomla-cms/14485</a>.</sub>

**Comment by bertmert** [comment]
I have tested this item :white_check_mark: successfully on 4e1763de2b340341e14cd7694bafe26ef4e46806<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14485">issues.joomla.org/tracker/joomla-cms/14485</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC as there are 2 successfully Tests?

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14485">issues.joomla.org/tracker/joomla-cms/14485</a>.</sub>

**Pull request merged**